### DISCUSSION TITLE
How to achieve this?

### DISCUSSION THREAD

**Discussion by huisky2000** [InteractiveComputerGraphics/SPlisHSPlasH#134] [Ideas]
Hi Professor,
This is a fantastic project in my view. I have been doing ship collisions impact mechanics and wondering can I use your library to build a simple visualized platform for simulating ship collisions? The collision detection might be an issue as it may have to be tri-mesh based (or others?)
I'd like to use an impulse-based collision algorithm, including the Coulomb friction factor and energy calculation.
I have limited knowledge of programming though, any chance to discuss further?
Thanks!

**Comment by janbender** [comment]
Currently the collisions between rigid objects are handled by the PBD library. However, it is also possible to exchange this by any other rigid body library. You just need a wrapper class like our PBDWrapper. Alternatively you could also extend the PBD library, which already provides a collision detection, by a new collision handling method.
